### ISSUE TITLE
Logging is broken in production on Windows

### ISSUE THREAD

**Issue by jamalex** [learningequality/ka-lite#5057]
## Summary

Logging to file was not working on Windows, as seen in #5030 -- seems likely to be multi-threaded concurrency issues (possibly `StreamHandler` being closed by one thread while still being used by another thread).

A temporary workaround to avoid bigger problems was introduced by https://github.com/learningequality/ka-lite/pull/5056, but it simply disables the `StreamHandler` output on Windows, thereby disabling logging.

We should remove the workaround introduced in #5056, and then sort out the original problem to get logging working properly again.
## Branch or installer method
- Branch: 0.16.x

**Comment by benjaoming** [comment]
I don't have a Windows dev environment so I'm likely the wrong person to assign.

However, this is a very critical bug troubleshooting windows deployment issues is made extremely difficult without proper logging.

**Comment by jamalex** [comment]
> I don't have a Windows dev environment so I'm likely the wrong person to assign.

I assigned you as you're our Django/logging expert; not sure who might be better to take it on (only people with Windows are frontend devs and interns), but feel free to attempt a re-assign. I did my initial debugging for this using a Virtualbox from: https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/linux/

**Comment by benjaoming** [comment]
Usual work-around (modify the paths in case you don't have Python installed in `C:\Python27`).

```
C:>cd C:\Python27
C:\Python27>cd Scripts
C:\Python27\Scripts>kalite start --foreground
```

**Comment by mrpau-richard** [comment]
Hi @benjaoming It's safe if we use the `kalite start --foreground` when starting the KA Lite server?
If that safe then we use it to Start the KA Lite server in the KA Lite windows application.

**Comment by benjaoming** [comment]
@mrpau-richard the solution we're looking for is to get the actual Python logging work so that logging is completely identical on all systems.

The above is just a work-around, not a fix :)

**Comment by benjaoming** [comment]
Fixed by @mrpau-richard in #5364
